--- a/cmake/BcToolboxCMakeUtils.cmake
+++ b/cmake/BcToolboxCMakeUtils.cmake
@@ -188,6 +188,8 @@
 		endif()
 
 		set(${OUTPUT_VERSION} "${full_version}" PARENT_SCOPE)
+	else()
+		set(${OUTPUT_VERSION} "${default_version}" CACHE STRING "")
 	endif()
 endfunction()
 
@@ -209,6 +211,8 @@
 		endif()
 
 		set(${OUTPUT_VERSION} "${snapshots_or_releases_state}" PARENT_SCOPE)
+	else()
+		set(${OUTPUT_VERSION} "${default_version}" CACHE STRING "")
 	endif()
 endfunction()
 
